diem-node/src/lib.rs

```diff
@@ -357,25 +358,25 @@ pub fn setup_environment(node_config: &NodeConfig, logger: Option<Arc<Logger>>)
             Some(&mut event_subscription_service),
         );
         let network_id = network_config.network_id.clone();
+        // Guarantee there is only one of this network
```

Comment:
I'm guessing this is something not currently verified?

Reply:
It is right now! below :D

Reply:
I meant "currently" as-in before this PR lands 😄

Reply:
Lol, nope I guess then